src/main/kotlin/org/openrewrite/RecipeMarkdownWriter.kt

```diff
@@ -604,15 +610,22 @@ import TabItem from '@theme/TabItem';
             //language=markdown
             writeln(
                 """
-                
+
                 ## Used by
-                
+
                 This recipe is used as part of the following composite recipes:
-                
+
                 """.trimIndent()
             )
             recipeContainedBy
-                .map { "* [${it.displayNameEscaped()}](/recipes/${getRecipePath(it)}.md)" }
+                .mapNotNull { recipe ->
+                    try {
+                        "* [${recipe.displayNameEscaped()}](/recipes/${getRecipePath(recipe)}.md)"
+                    } catch (e: RuntimeException) {
+                        System.err.println("Warning: Could not generate path for recipe ${recipe.name}: ${e.message}")
+                        null
+                    }
+                }
```

Comment:
When we ran into a recipe with a package we didn't recognize - it would break stuff - this lets it continue so the entire process isn't destroyed because of one package (like `com.google.*`)